Update deploy-two-node-clustered-file-server.md (#3854)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

- **Network Adapters and cable:** The network hardware, like other components in the failover cluster solution, must be compatible with Windows Server 2016 or Windows Server 2019. If you use iSCSI, the network adapters must be dedicated to either network communication or iSCSI, not both. In the network infrastructure that connects your cluster nodes, avoid having single points of failure. There are multiple ways of accomplishing this. You can connect your cluster nodes by multiple, distinct networks. Alternatively, you can connect your cluster nodes with one network that is constructed with teamed network adapters, redundant switches, redundant routers, or similar hardware that removes single points of failure.

   > [!NOTE]
   > If the cluster nodes are connected with a single network, the network passes the redundancy requirement in the Validate a Configuration wizard.  However, the report includes a warning that the network should not have a single point of failure.

- **Device Controllers or appropriate adapters for storage:**

>     - **Serial Attached SCSI or Fibre Channel:** If you are using Serial Attached SCSI or Fibre Channel, in all clustered servers, all components of the storage stack should be identical. It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components be identical.  It is recommended that the mass-storage device controllers—that is, the host bus adapter (HBA), HBA drivers, and HBA firmware—that are attached to cluster storage be identical. If you use dissimilar HBAs, you should verify with the storage vendor that you are following their supported or recommended configurations.

Reviewer: This use of "be" looks archaic/poetic or even like some form of broken English:
 ```suggestion
    - **Serial Attached SCSI or Fibre Channel:** If you are using Serial Attached SCSI or Fibre Channel, in all clustered servers, all components of the storage stack should be identical. It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components are identical.  It is recommended that the mass-storage device controllers—that is, the host bus adapter (HBA), HBA drivers, and HBA firmware—that are attached to cluster storage be identical. If you use dissimilar HBAs, you should verify with the storage vendor that you are following their supported or recommended configurations.
```
Suggestion: [It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components **are** identical.]
(If you wanted to use "be", it could have been written as follows: [The requirement for the multipath I/O (MPIO) software and Device Specific Module (DSM) software components is that they **need to be** identical.] )

Reviewer: BTW, the section I have made this suggestion for is part of the previously unchanged document.

- **Clients:** As needed for testing, you can connect one or more networked clients to the failover cluster that you create, and observe the effect on a client when you move or fail over the clustered file server from one cluster node to the other.

> - **Account for administering the cluster:** When you first create a cluster or add servers to it, you must be logged on to the domain with an account that has administrator rights and permissions on all servers in that cluster. The account does not need to be a Domain Admins account, but can be a Domain Users account that is in the Administrators group on each clustered server. In addition, if the account is not a Domain Admins account, the account (or the group that the account is a member of) must be given the **Create Computer Objects** and **Read All Properties** permissions in the domain organizational unit (OU) that is to reside in.

Reviewer: incomplete sentence ending (readability flow):
```suggestion
- **Account for administering the cluster:** When you first create a cluster or add servers to it, you must be logged on to the domain with an account that has administrator rights and permissions on all servers in that cluster. The account does not need to be a Domain Admins account, but can be a Domain Users account that is in the Administrators group on each clustered server. In addition, if the account is not a Domain Admins account, the account (or the group that the account is a member of) must be given the **Create Computer Objects** and **Read All Properties** permissions in the domain organizational unit (OU) that it is going to reside in.
```
```
[...] that it is going to reside in.
```

Author: Picked that up - thanks.

Author: I've changed it so as not to end in a preposition! :-)

3. If the **Before you begin** window opens, choose **Next**.

4. In the **Select Servers or a Cluster** window, add in the names of the two machines that are to be the nodes of the cluster.  For example, if the names are NODE1 and NODE2, enter the name and select **Add**.  You can also choose the **Browse** button to search Active Directory for the names.  Once both are listed under **Selected Servers**, choose **Next**.

5. In the **Testing Options** window, select **Run all tests (recommended)**, and **Next**.

> 6. On the **Confirmation** page, it gives you the listing of all the tests it it is check.  Choose **Next** and the tests begin.

Reviewer: oh hello ... I was too quick on the OK button:
```suggestion
6. On the **Confirmation** page, it gives you the listing of all the tests it checks.  Choose **Next** and the tests begin.
```

3. If the **Before you begin** window opens, choose **Next**.

4. In the **Select Servers** window, add in the names of the two machines that are to be the nodes of the cluster.  For example, if the names are NODE1 and NODE2, enter the name and select **Add**.  You can also choose the **Browse** button to search Active Directory for the names.  Once both are listed under **Selected Servers**, choose **Next**.

> 5. In the **Access Point for Administering the Cluster** window, input the name of the cluster you are using.  Please note that this is not the name you use to connect to your file shares with.  This is for simply administrating the cluster.

Reviewer: I recommend a more natural speech form for the last sentence here:
```suggestion
5. In the **Access Point for Administering the Cluster** window, input the name of the cluster you are using.  Please note that this is not the name you use to connect to your file shares with.  This name is simply for administrating the cluster.
```

   ![File Server Type](media/Cluster-File-Server/Cluster-FS-File-Server-Type.png)

8. In the **Client Access Point** window, input the name of the file server you are using.  Please note that this is not the name of the cluster.  
This is for the file share connectivity.  

> For example, if you want to connect to \\\\SERVER, specify the name as SERVER.

Reviewer: add an extra space, just to be on the safe side (rich diff shows no spacing here):
```suggestion
For example, if you want to connect to  \\\\SERVER, specify the name as SERVER.
```

   > [!NOTE]
   > If you are using static IP addresses, you need to select the network to use and input the IP address to be used for the cluster name.  If you are using DHCP for your IP addresses, the IP address is configured automatically for you.

6. Choose **Next**.

7. In the **Select Storage** window, select the additional drive (not the witness) that holds your shares and **Next**.

8. On the **Confirmation** page, verify your configuration and select **Next**.

9. On the **Summary** page, it gives you the configuration it has created.  You can select View Report to see the report of the file server role creation.

10. Under **Roles** in the console tree, you see the new role you created listed as the name you created.  With it highlighted, under the **Actions** pane on the right, choose **Add a share**.

11. Run through the share wizard inputting the following:

>     - Type of share it is

Reviewer: Simplification (in contrast to expanding it to a natural sentence):
```suggestion
    - Type of share
```
Alternative: [The type of share it is]

    - Location/path of the folder shared to be shared
    - The name of the share users connect to
    - Additional settings such as Access-based enumeration, caching, encryption, etc

>     - File level permissions if they are ptjer than the defaults

Reviewer: typo
```suggestion
    - File level permissions if they are other than the defaults
```

   ![File Server Type](media/Cluster-File-Server/Cluster-FS-File-Server-Type.png)

8. In the **Client Access Point** window, input the name of the file server you will be using.  Please note that this is not the name of the cluster.  
This is for the file share connectivity.  
For example, if you want to connect to \\\\SERVER, specify the name as SERVER.

   > [!NOTE]

>    > If you are using static IP Addresses, you will need to select the network to use and input the IP Address it will use for the cluster name.  If you are using DHCP for your IP Addresses, the IP Address will be configured automatically for you.

Reviewer: Thank you for the positive reply. May I ask you to make these ones lowercase too? (or you can merge my suggestion, if you prefer)
```suggestion
   > If you are using static IP addresses, you will need to select the network to use and input the IP address it will use for the cluster name.  If you are using DHCP for your IP addresses, the IP address will be configured automatically for you.
```
